src/middleware/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Request, Response, NextFunction } from 'express'
+import { ValidationFailedError, InvalidPathError } from './../errors'
+
+export default (err: Error, _req: Request, res: Response, _next: NextFunction): void => {
+    let statusCode = 500
+    const message = err.message || 'Something went wrong'
+    if (err instanceof ValidationFailedError || InvalidPathError) statusCode = 400
```

Comment:
The conditional expression will return `true` or `InvalidPathError`, thus the condition will always be true and statusCode will always be 400. Change it to:
```js
if (err instanceof ValidationFailedError || err instanceof InvalidPathError)
```